dataland-frontend/tests/component/components/forms/parts/elements/derived/ProcurementCategoryFormElement.cy.ts

```diff
@@ -14,6 +14,7 @@ describe("Component tests for the CreateLksgDataset that test dependent fields",
         };
       },
     }).then(() => {
+      cy.get('[data-test="dataPointToggleButton"]').click();
```

Comment:
You must decide if you prefer this row or change isItActive to isActive in the data block above (otherwise remove it). In any case action required.

Reply:
done